Childrens-Social-Care-CPD/Program.cs

```diff
@@ -32,6 +34,10 @@
 var options = new ApplicationInsightsServiceOptions { ConnectionString = "InstrumentationKey=13cbca44-e50d-4444-ab56-6c151cad789d;IngestionEndpoint=https://westeurope-5.in.applicationinsights.azure.com/;LiveEndpoint=https://westeurope.livediagnostics.monitor.azure.com/" };
 builder.Services.AddApplicationInsightsTelemetry(options: options);
 
+var telemetryConfiguration = TelemetryConfiguration.CreateDefault();
```

Comment:
You are setting up telemetryConfiguration, however not using it so it wont work,  we can use something like this

var options = new ApplicationInsightsServiceOptions { ConnectionString = Environment.GetEnvironmentVariable(SiteConstants.CPD_INSTRUMENTATION_KEY) };
builder.Services.AddApplicationInsightsTelemetry(options: options);

Reply:
Also, AL needs to pass connection string in CPD_INSTRUMENTATION_KEY  env variable

Reply:
CPD_INSTRUMENTATION_KEY is already being passed in the environment.